model/src/main/scala/scalasvg/element/Circle.scala

```diff
@@ -0,0 +1,23 @@
+package scalasvg.element
+
+import scalasvg.attribute.generic.{Core, Styling, ConditionalProcessing}
+import scalasvg.element.category.Category
+
+/**
+  * The <circle> SVG element is an SVG basic shape, used to draw circles based on a center point and a radius.
+  * @param cx The x-axis coordinate of the center of the circle.
+  * @param cy The y-axis coordinate of the center of the circle.
+  * @param r The radius of the circle. A value lower or equal to zero disables rendering of the circle.
+  * @param pathLength The total length for the circle's circumference, in user units.
+  */
+final case class Circle(cx: String, cy: String, r: String, pathLength: Option[Number]=None, attributes: Seq[Circle.Attribute])
```

Comment:
I suggest we split all the parameters into three groups:
* element-specific attributes, such as `cx` or `xy` for `circle`
* a var-arg of all other applicable attributes
* content, when applicable

So, in this case, it would be:
```scala
final case class Circle(cx: String, cy: String, r: String, pathLength: Option[Number] = None)
                       (attributes: Circle.Attribute*)
  extends Element.Empty
  with Category.BasicShape
```

Reply:
Done. But now there are two groups of arguments.